spec/javascripts/helpers/test-coverage.mjs

```diff
@@ -0,0 +1,9 @@
+window.__coverage__ ??= {};
+
+class TestCoverageReporter {
+  static suiteDone(result) {
```

Comment:
Could we add a code comment here, something along the lines of:
`Runs after every top level Jasmine describe-block`

Reply:
Done